packages/orchestration/src/examples/unbond.flows.js

```diff
@@ -24,13 +24,12 @@ export const unbondAndLiquidStake = async (
   _offerArgs,
 ) => {
   console.log('zcf within the membrane', zcf);
-  // We would actually alreaady have the account from the orchestrator
-  // ??? could these be passed in? It would reduce the size of this handler,
-  // keeping it focused on long-running operations.
-  const omni = await orch.getChain('omniflixhub');
-  const omniAccount = await omni.makeAccount();
+  // Osmosis is one of the few chains with icqEnabled
+  const osmosis = await orch.getChain('osmosis');
```

Comment:
`omniflix` appears to have icq enabled, but that's not to suggest we should revert this change. `osmosis` is preferable from my POV since it's integrated in `multichain-testing` - starship doesn't yet support `omniflix`.